app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
```

Comment:
Put a space after namespace. It's important to separate unrelated code so that it is easy to read.

Reply:
I edited it.